contentcuration/contentcuration/frontend/channelEdit/views/NodePanel.vue

```diff
@@ -95,6 +101,30 @@
     },
     methods: {
       ...mapActions('contentNode', ['loadChildren']),
+      goToNodeDetail(nodeId) {
+        this.$router.push({
+          name: RouterNames.TREE_VIEW,
+          params: {
+            nodeId: this.parentId,
+            detailNodeId: nodeId,
+          },
+        });
+      },
+      goToTopic(topicId) {
+        this.$router.push({
+          name: RouterNames.TREE_VIEW,
+          params: {
+            nodeId: topicId,
+          },
+        });
+      },
+      onNodeDoubleClick(node) {
+        if (node.kind === ContentKindsNames.TOPIC) {
+          this.goToTopic(node.id);
+        } else {
+          this.goToNodeDetail(node.id);
+        }
+      },
```

Comment:
Thanks for adding this, it was on my list. I did have it at one point but got thrown out with me separating draggable I think.

Reply:
Sure sure :)